plugins/de.fraunhofer.ipa.rossystem.xtext/src/de/fraunhofer/ipa/rossystem/RosSystem.xtext

```diff
@@ -77,40 +83,39 @@ InterfaceReference_Impl returns InterfaceReference:
     ;
 
 RosPublisherReference returns RosPublisherReference:
-    'fromPub:'From=[ros::Publisher|EString]
+    'fromPub:'from=[ros::Publisher|EString]
 ;
 
 RosSubscriberReference returns RosSubscriberReference:
-    'fromSub:'From=[ros::Subscriber|EString]
+    'fromSub:'from=[ros::Subscriber|EString]
 ;
 
 RosServiceServerReference returns RosServiceServerReference:
-    'fromSS:'From=[ros::ServiceServer|EString]
+    'fromSS:'from=[ros::ServiceServer|EString]
 ;
 
 RosServerClientReference returns RosServerClientReference:
-    'fromSC:'From=[ros::ServiceClient|EString]
+    'fromSC:'from=[ros::ServiceClient|EString]
 ;
 
 RosActionServerReference returns RosActionServerReference:
-    'fromAS:'From=[ros::ActionServer|EString]
+    'fromAS:'from=[ros::ActionServer|EString]
 ;
 
 RosActionClientReference returns RosActionClientReference:
-    'fromAC:'From=[ros::ActionClient|EString]
+    'fromAC:'from=[ros::ActionClient|EString]
 ;
 
 RosParameter returns RosParameter:
-    Name=EString':'
+    PreListElement name=EString':' from=[ros::Parameter|EString]
```

Comment:
PreListElement was defined in another commit, I will remove it from here. This commit is about uncapitalize the definition of the attributes